package.json

```diff
@@ -33,19 +33,19 @@
   "homepage": "https://github.com/dreipol/vue-ui#readme",
   "devDependencies": {
     "@babel/core": "^7.1.2",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
     "@babel/preset-env": "^7.1.0",
     "@dreipol/eslint-config": "^6.0.2",
-    "@dreipol/eslint-config-vue": "^1.0.0",
-    "@vue/test-utils": "^1.0.0-beta.28",
+    "@dreipol/eslint-config-vue": "^2.0.2",
+    "@vue/test-utils": "^1.0.0-beta.29",
```

Comment:
downgrade to `1.0.0-beta.28` and the tests should work again. I think they have broken something

Reply:
Awesome hint! That saved me a lot of time.